### DISCUSSION TITLE
PromQL can't work on grafana variables query filed

### DISCUSSION THREAD

**Discussion by J-lena** [grafana/grafana#46923] [Platform]
hi , why PromQL can't work on Grafana variables query filed, here is my prometheus query expression:

group(idelta(kindling_topology_request_duration_nanoseconds_count{dst_namespace="bookdemo"}[30m]) ) by (src_namespace,src_workload_name,dst_namespace,dst_workload_name)or group (idelta(kindling_topology_request_duration_nanoseconds_count{src_namespace="bookdemo"}[30m])) by (src_namespace,src_workload_name,dst_namespace,dst_workload_name).

but in prometheus ,this expression can run successfully.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
